src/App/Fossa/Container/AnalyzeNative.hs

```diff
@@ -44,5 +47,26 @@ analyzeExperimental ContainerAnalyzeConfig{..} = do
     ContainerOCIRegistry _ _ -> fatalText "container images from oci registry are not yet supported!"
     ContainerExportedTarball _ -> fatalText "exported container images are not yet supported!"
 
-parseContainerImageSource :: (Has (Lift IO) sig m) => Text -> m (ContainerImageSource)
-parseContainerImageSource tag = pure $ ContainerDockerImage tag
+parseContainerImageSource ::
+  ( Has (Lift IO) sig m
+  , Has Diagnostics sig m
+  , Has ReadFS sig m
+  ) =>
+  Text ->
+  m (ContainerImageSource)
+parseContainerImageSource = parseExportedTarballSource
+
+parseExportedTarballSource ::
```

Comment:
Are these functions not the exact same? Why create `parseExportedTarballSource`?

Reply:
yep, adding scaffolding now, so we can easily add `parseExportedTarballSource  <||> parseDockerDaemonSource <||> parseOCIRegistrySource` in subsequent PRs.